Readme.md

```diff
@@ -1,40 +1,39 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128590153/22.2.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1393)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [ActionsInvokerViewController.cs](./CS/EFCore/ExecuteActionEF/ExecuteActionEF.Module/Controllers/ActionInvokerViewController.cs) 
-<!-- default file list end -->
-# How to execute Actions in code
+# XAF - How to execute Actions programmatically
 
-## Scenario
+This example explains how you can trigger the code inside `Execute` event handlers of custom or standard XAF Actions. For example, you can programmatically invoke an Action that shows a View or runs code in the background.
 
-Normally, XAF Actions are executed only by end users using the interaction with UI elements (for instance, button clicks, navigation, or main menu selection). In advanced UI scenarios, XAF developers may want to programmatically trigger the code inside the Execute event handlers of custom or standard XAF Actions. For instance, a XAF developer wants to execute default Action logic that shows a View or runs some code in the background.
+> **NOTE** 
+> We do not recommend this approach. Refactor your Action's event handlers and extract required code into separate methods. Call these separate methods directly without triggering UI-related Action code.
```

Comment:
We do not recommend this approach. Refactor your Action's event handlers and extract the code into separate methods. Call these separate methods directly without triggering UI-related Action code.